Source/BSN.IpTables.Cli/custom/Connect-BsnIPTablesCli.ps1

```diff
@@ -0,0 +1,22 @@
+function Connect-BsnIPTablesCli {
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory)]
+        [BSN.IpTables.V1.Category('Uri')]
+        [System.String]
+        # Target Server Address
+        ${ServerAddress}
+    )
+
+    begin {
+        # Save the ServerAddress in a session variable
+        $env:ServerAddress = $ServerAddress
```

Comment:
I think it is good to check env variables does not fill before you set, and make warning if this variable does not empty

Reply:
We could add a message to warn server address change.

Reply:
Done.